en/dronecan/ark_flow.md

```diff
@@ -97,46 +101,6 @@ Parameter | Description
 --- | ---
 <a id="CANNODE_TERM"></a>[CANNODE_TERM](../advanced_config/parameter_reference.md#CANNODE_TERM) | CAN built-in bus termination.
 
-
-## Building Ark Flow Firmware
```

Comment:
Ah, so this has been pushed into "PX4 DroneCAN Firmware". That works.

Reply:
Yes. I thought we should keep things in a central place instead of repeating information.